### ISSUE TITLE
support images in actual dashboard markdown widgets

### ISSUE THREAD

**Issue by jpvajda** [newrelic/newrelic-observability-packs#235]
We had a case where a contributor wanted to added images to be used in a dashboard's markdown widget as helper files. we don't currently support that but it be nice to allow a contributor to do this.

**Comment by jpvajda** [comment]
We seemed to support this in several cases, here is one for [Python](https://github.com/newrelic/newrelic-quickstarts/tree/main/quickstarts/python/python) So is there anything to do here? @aswanson-nr ?

**Comment by aswanson-nr** [comment]
@jpvajda Yep! This should be totally doable. We could add an `images` subdirectory to the `dashboards` directory for images that are going to be referenced in the dashboard's markdown. That would require us to update the checks we currently have for images to account for this change and we will need to work with @jbeveland27 to ensure they aren't pulled in as screenshots.
